src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -42,27 +40,27 @@ class Ledger(vm: VM) extends Logger {
     * This function runs transaction
     *
     * @param block
+    * @param blockchain
     * @param storages
-    * @param stateStorage
+    * @param validators
     */
   private def executeBlockTransactions(
     block: Block,
     blockchain: Blockchain,
     storages: BlockchainStorages,
-    stateStorage: NodeStorage):
+    validators: Validators):
```

Comment:
This looks like something that we could put inside the config I am working on. If we do that, we won't have to check for homestead block in signedTransactionValidator anymore (we could just use different instances in configs for different blocks). Do you think it makes sense?